crates/jit/src/unwind/systemv.rs

```diff
@@ -25,6 +25,12 @@ impl UnwindRegistration {
         unwind_info: *mut u8,
         unwind_len: usize,
     ) -> Result<UnwindRegistration> {
+        debug_assert_eq!(
+            unwind_info as usize % 0x1000,
```

Comment:
I think that this `0x1000` can probably be `region::page::size()` here in this crate